### PR TITLE
onboarding: ask for invite to Percy and Chromatic

### PR DESCRIPTION
**Pull request by tjkandala**
Question: what do we call software engineers who work on the frontend?  Common titles: frontend developer, web developer, frontend engineer, software engineer - frontend. We use "software engineer - frontend" on the hiring page, but that wouldn't sound right here ("Ask a software engineer - frontend").

Edit: Added Chromatic as well

### CODE REVIEW

File: handbook/engineering/onboarding.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ Your manager should complete the following steps when you join:
   - [HoneyComb.io](https://www.honeycomb.io/)
   - Ask Christina to send an invite to [Productboard](https://sourcegraph.productboard.com)
   - Ask a member of the Design team to invite as "Viewer" to [Figma](https://figma.com)
+  - Ask a frontend engineer to invite you to [Percy](https://percy.io/) (optional; recommended for frontend engineers)
+  - Ask a frontend engineer to invite you to [Chromatic](https://www.chromatic.com/) (optional; recommended for frontend engineers)
```
Comment (nicksnyder) [inline handbook/engineering/onboarding.md:35]: This is the EM section, so this should probably be under "Grant access to necessary services".

Is this applicable to all engineers? Should there be web specific onboarding section?

Comment (tjkandala) [reply handbook/engineering/onboarding.md:35]: I originally intended to add this to the web team page, but I noticed that non-web engineers were unable to approve small UI changes on Percy or Chromatic. If we made these services visible during onboarding, they wouldn't have to wait for frontend engineers' approval for changes to copy. 

I also need to formalize + document the UI review process. Search for "chromatic" in the handbook returns 1 real result, which is the Product team onboarding page.

Comment (tjkandala) [reply handbook/engineering/onboarding.md:35]: Inspired by this thread: https://sourcegraph.slack.com/archives/C07KZF47K/p1600439726004300

Comment (felixfbecker) [reply handbook/engineering/onboarding.md:35]: Chromatic uses GitHub permissions, so afaik there is no way or necessity to "invite" people. Agree though that we should document the review process more. The idea here was that the @sourcegraph/design team has the authority over the UI review, while engineers have the authority over the code review. Engineers may stamp small/obvious changes too though (just like with code).

**Pull request closed without merging**